src/pages/Troubleshooting.vue

```diff
@@ -50,15 +57,19 @@
 
 <script>
 import TroubleshootingLineItem from '@/components/TroubleshootingLineItem.vue';
+import Checkbox from '@/components/form/Checkbox';
 
 const { ipcRenderer } = require('electron');
 const K8s = require('../k8s-engine/k8s');
 
 export default {
   name:       'Troubleshooting',
   title:      'Troubleshooting',
-  components: { TroubleshootingLineItem },
-  data:       () => ({ state: ipcRenderer.sendSync('k8s-state') }),
+  components: { TroubleshootingLineItem, Checkbox },
+  data:       () => ({
+    state:    ipcRenderer.sendSync('k8s-state'),
```

Comment:
Please talk to Everton on how to do this asynchronously. He's got a PR on this that's waiting on git actions before it gets merged